docs/setup/advanced/settings.md

```diff
@@ -0,0 +1,76 @@
+# Settings
+
+Settings allow you to configure system-wide behavior of the eWallet server. All settings are configurable via the Admin Panel.
+
+## General
+
+Below are the general settings needed for the eWallet to run.
+
+- `base_url`: The base to use when building URLs.
+
+## Incoming Requests
+
+The eWallet allows you to configure different aspects of the incoming requests.
+We try to provide sane default values, but if needed, the following environment variables are configurable:
```

Comment:
There are still mentions of `environment variable` in this doc, we should switch all those to `settings`.